presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -28,6 +28,9 @@
     private static final String TASK_HASH_BUILD_CONCURRENCY = "task_hash_build_concurrency";
     private static final String TASK_AGGREGATION_CONCURRENCY = "task_aggregation_concurrency";
     private static final String QUERY_MAX_MEMORY = "query_max_memory";
+    private static final String VERBOSE_STATS = "verbose_stats";
+    private static final String TASK_MAX_MEMORY = "task_max_memory";
```

Comment:
I'm planning to remove task.max-memory soon, in favor of query.max-memory-per-node, so I don't think we should add a session property for it. Same for the big query task memory